scripts-available/CDB_TableMetadata.sql

```diff
@@ -223,15 +214,39 @@ CREATE OR REPLACE FUNCTION public.CDB_TableMetadataTouch(tablename regclass)
     RETURNS void AS
```

Comment:
Who else calls this function and what's the impact on this change to them?

Reply:
This is called by the editor as well - I tested all workflows of updating data through an org user account which call this function and those work as expected.